### PR TITLE
[WEBDEV-590] Add assets to JSON

### PR DESCRIPTION
**Pull request by cazwazacz**
- Add Pugin version, favicon URL and OpenSearch URL to JSON

### CODE REVIEW

File: spec/fixtures/serializers/component_serializer/list_component_serializer/fixture.yml [hunk 1]
```diff
@@ -1,23 +1,60 @@
 ---
```
Comment (cazwazacz) [inline spec/fixtures/serializers/component_serializer/list_component_serializer/fixture.yml]: Delete unused fixture

File: spec/fixtures/serializers/component_serializer/list_component_serializer/list--pipe.yml [hunk 2]
```diff
@@ -1,23 +1,60 @@
 ---
```
Comment (cazwazacz) [inline spec/fixtures/serializers/component_serializer/list_component_serializer/list--pipe.yml]: Delete unused fixture

File: spec/fixtures/serializers/component_serializer/status_highlight_component_serializer/fixture.yml [hunk 3]
```diff
@@ -5,14 +5,40 @@ data:
   components:
```
Comment (cazwazacz) [inline spec/fixtures/serializers/component_serializer/status_highlight_component_serializer/fixture.yml]: Delete unused fixture

File: app/serializers/page_serializer/base_page_serializer.rb [hunk 4]
```diff
@@ -1,28 +1,57 @@
 module PageSerializer
   # The serializer which all page serializers inherit from.
   class BasePageSerializer < BaseSerializer
+    FAVICON_URL = 'https://s3-eu-west-1.amazonaws.com/web1live.static-assets/favicon.ico'.freeze
```
Comment (mattrayner) [inline app/serializers/page_serializer/base_page_serializer.rb]: NOPE

File: app/serializers/page_serializer/base_page_serializer.rb [hunk 5]
```diff
@@ -1,28 +1,57 @@
...
+    OPENSEARCH_URL = 'http://beta.parliament.uk/search/opensearch'.freeze
```
Comment (mattrayner) [inline app/serializers/page_serializer/base_page_serializer.rb]: This should be an environment variable

Comment (mattrayner) [reply app/serializers/page_serializer/base_page_serializer.rb]: use the rails route

**Pull request merged**